source/educators/how-tos/course_development/exercise_tools/add_annotation.rst

```diff
@@ -91,4 +91,12 @@ To add the **Annotation problem** segment of the problem, follow these steps.
            </solution>
          </problem>
 
-#. Select **Save**.
\ No newline at end of file
+#. Select **Save**.
+
+.. seealso::
+ :class: dropdown
+
+ :ref:`Calculator Tool` (how-to)
+
+ :ref:`Chemical Equation Problem` (how-to)
```

Comment:
```
/home/docs/checkouts/readthedocs.org/user_builds/docsopenedxorg/checkouts/698/source/educators/how-tos/course_development/exercise_tools/add_annotation.rst:99: WARNING: undefined label: 'calculator tool'
/home/docs/checkouts/readthedocs.org/user_builds/docsopenedxorg/checkouts/698/source/educators/how-tos/course_development/exercise_tools/add_annotation.rst:101: WARNING: undefined label: 'chemical equation problem'
```

What are you trying to reference here? You need to add the reference labels in the appropriate documents.